servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/RetryingHttpRequesterFilter.java

```diff
@@ -530,7 +529,7 @@ public static BackOffPolicy ofExponentialBackoffDeltaJitter(
         private boolean waitForLb = true;
         private boolean ignoreSdErrors;
 
-        private int maxRetries = MAX_VALUE;
+        private int maxRetries = 5;
```

Comment:
This is the default TOTAL max retries. The individual back-off policies have separate max-retries (eg. 3 for retryable exceptions)

Reply:
🤦 I forgot about that, nvm